Makefile.defs

```diff
@@ -64,6 +64,9 @@ GO_TAGS_FLAGS = osusergo
 # RACE is specified.
 GOTEST_COVER_OPTS =
 
+GOLANGCILINT_WANT_VERSION = 1.44.2
```

Comment:
```
GOLANGCILINT_IMAGE_SHA = sha256:c61216e0aeb8b38dcdcf790e2b7eee6e2f8c5564bd5f938ce09f7a027a409d56
```

(this SHA is for the v1.45.0 so it can be added in the next commit)

Reply:
v1.45.2 is already out.

Reply:
Updated to v1.45.2 (SHA `e84b639c061c8888be91939c78dae9b1525359954e405ab0d9868a46861bd21b`)